crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,14 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    case service_name
+    when :chef
+      init_cmd = ChefObject.get_service_init_cmd(@node["platform"], @node["platform_version"], "chef-client", action)
+    end
+    ssh_cmd(init_cmd) unless init_cmd.nil?
```

Comment:
> And what happens on Hyper-V hosts?

i think this should be fixed in another place. I am not sure how `apply_role` works for a HyperV node but `run_remote_chef_client` which is executed during `apply_role` goes over ssh as well, so i think this here should not be a problem on the HyperV host.